docs/src/modules/java/pages/eventsourced.adoc

```diff
@@ -61,12 +61,10 @@ include::example$java-eventing-shopping-cart/src/main/java/shopping/cart/Shoppin
 
 == Emitting events
 
-Commands can modify state by emitting events. A command handler can emit an event by taking in a link:{attachmentsdir}/api/com/akkaserverless/javasdk/eventsourcedentity/CommandContext.html[`CommandContext` {tab-icon}, window="new"] parameter, and invoking the link:{attachmentsdir}/api/com/akkaserverless/javasdk/eventsourcedentity/CommandContext.html#emit(java.lang.Object)[`emit` {tab-icon}, window="new"] method on it. Invoking `emit` will immediately invoke the associated event handler for that event. This validates that the event can be applied to the current state, and also updates state for command handler processing.
+Commands can modify state by emitting events. A command handler can emit an event by taking in a link:{attachmentsdir}/api/com/akkaserverless/javasdk/eventsourcedentity/CommandContext.html[`CommandContext` {tab-icon}, window="new"] parameter, and invoking the link:{attachmentsdir}/api/com/akkaserverless/javasdk/eventsourcedentity/CommandContext.html#emit(java.lang.Object)[`emit` {tab-icon}, window="new"] method on it. 
```

Comment:
May be useful to keep the information about event handlers being invoked immediately. Adding back this text, extended a little:

```suggestion
Commands can modify state by emitting events. A command handler can emit an event by taking in a link:{attachmentsdir}/api/com/akkaserverless/javasdk/eventsourcedentity/CommandContext.html[`CommandContext` {tab-icon}, window="new"] parameter, and invoking the link:{attachmentsdir}/api/com/akkaserverless/javasdk/eventsourcedentity/CommandContext.html#emit(java.lang.Object)[`emit` {tab-icon}, window="new"] method on it.

Invoking `emit` will immediately invoke the associated event handler for that event. This validates that the event can be applied to the current state, and also makes the updated state available for creating the response message or any other subsequent processing in the current command handler.